packages/table/src/table.tsx

```diff
@@ -1185,6 +1190,86 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         const newSelectionRegions = [Regions.cell(newFocusedCell.row, newFocusedCell.col)];
         this.handleSelection(newSelectionRegions);
         this.handleFocus(newFocusedCell);
+
+        // keep the focused cell in view
+        this.scrollBodyToFocusedCell(newFocusedCell);
+    }
+
+    private scrollBodyToFocusedCell = (focusedCell: IFocusedCellCoordinates) => {
+        if (focusedCell == null) {
```

Comment:
Whoops, this check is redunant with [Line 1159](https://github.com/palantir/blueprint/pull/1023/files#diff-6bc89efba539cc29af0dae9b17c57f75R1159) now. Will remove.